snapcraft/plugins/rust.py

```diff
@@ -31,6 +31,9 @@
     - rust-features
       (list of strings)
       Features used to build optional dependencies
+    - libc:
+      (string; default: gnu)
+      Libc to link against. Valid options are: gnu (glibc) and musl
```

Comment:
I don't have much clue about this, but why call it gnu instead of glibc?

Reply:
`gnu` and `musl` are the names of the toolchains. I mention glibc there to clarify what it means.